govcd/lbservicemonitor_test.go

```diff
@@ -0,0 +1,69 @@
+// +build lb lbServiceMonitor functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_LBServiceMonitor(check *C) {
```

Comment:
Please add doc what this test is doing/purpose

Reply:
Done